modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1385,6 +1386,188 @@ Verify that this is the intended server.
+----
+
+Full title of the returned code::
+`CREATE TEXT INDEX myIndex IF NOT EXISTS FOR ()-[e:REL_TYPE]-() ON (e.property)` has no effect.
+
+Full description of the returned code::
+`RANGE INDEX myIndex FOR (e:Label) ON (e.property)` already exists.
+
+Suggestions for improvement::
+Chose a different name for the new index and try again as the existing index was not similar to the one attempting to be created.
++
+[source, cypher]
+----
+CREATE TEXT INDEX myIndex2 IF NOT EXISTS FOR ()-[r:REL_TYPE]-() ON (r.property)
+----
+
+====
+
+.Creating a constraint when an identical constraint already exists
+====
+Given a node key constraint on `(:Label \{property})` named `nodeKeyLabelPropertyConstraint`.
+
+Command::
++
+[source, cypher]
+----
+CREATE CONSTRAINT nodeKeyLabelPropertyConstraint IF NOT EXISTS FOR (n:Label) REQUIRE (n.property) IS NODE KEY
+----
+
+Full title of the returned code::
+`CREATE CONSTRAINT nodeKeyLabelPropertyConstraint IF NOT EXISTS FOR (e:Label) REQUIRE (e.property) IS NODE KEY` has no effect.
+
+Full description of the returned code::
+`CONSTRAINT nodeKeyLabelPropertyConstraint FOR (e:Label) REQUIRE (e.property) IS NODE KEY` already exists.
+
+Suggestions for improvement::
+The existing constraint is likely what was wanted, no further actions needed.
+
+====
+
+.Creating a constraint when another unrelated constraint using that name already exists
+====
+Given a node key constraint on `(:Label \{property})` named `myConstraint`.
+
+Command::
++
+[source, cypher]
+----
+CREATE CONSTRAINT myConstraint IF NOT EXISTS FOR (n:Label2) REQUIRE (n.property2) IS NOT NULL
+----
+
+Full title of the returned code::
+`CREATE CONSTRAINT myConstraint IF NOT EXISTS FOR (e:Label2) REQUIRE (e.property2) IS NOT NULL` has no effect.
+
+Full description of the returned code::
+`CONSTRAINT myConstraint FOR (e:Label) REQUIRE (e.property) IS NODE KEY` already exists.
+
+Suggestions for improvement::
+Chose a different name for the new constraint and try again as the existing constraint was not similar to the one attempting to be created.
++
+[source, cypher]
+----
+CREATE CONSTRAINT myConstraint2 IF NOT EXISTS FOR (n:Label2) REQUIRE (n.property2) IS NOT NULL
+----
+
+====
+
+[#_neo_clientnotification_schema_indexorconstraintdoesnotexist]
+=== IndexOrConstraintDoesNotExist
+
+.Notification details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Schema.IndexOrConstraintDoesNotExist
+|Title
+a|`<command>` has no effect.
+|Description
+a|`<name>` does not exist.
+|Severity
+m|INFORMATION
+|Category
+m|SCHEMA
+|===
+
+.Attempting to drop a non-existing index
+====
+
+Command::
++
+[source, cypher]
+----
+DROP INDEX nonExistingIndex IF EXISTS
+----
+
+Full title of the returned code::
+`DROP INDEX nonExistingIndex IF EXISTS` has no effect.
+
+Full description of the returned code::
+`nonExistingIndex` does not exist.
+
+Suggestions for improvement::
+Verify that this is the intended index and that it is spelled correctly.
```

Comment:
the two drop ones I think are fine, they are basically taken from the revoke notifications above.